MODULE.bazel

```diff
@@ -42,13 +72,4 @@ bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.7.1", dev_dependenc
 bazel_dep(name = "aspect_bazel_lib", version = "2.8.1", dev_dependency = True)
 bazel_dep(name = "buildifier_prebuilt", version = "6.4.0", dev_dependency = True)
 bazel_dep(name = "rules_pkg", version = "1.0.1", dev_dependency = True)
-bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True)
-bazel_dep(name = "rules_proto", version = "6.0.0", dev_dependency = True)
-bazel_dep(name = "rules_java", version = "7.9.0", dev_dependency = True)
-bazel_dep(name = "protobuf", version = "27.3", dev_dependency = True)
-bazel_dep(name = "toolchains_protoc", version = "0.3.2", dev_dependency = True)
-
-register_toolchains(
-    "//tools/toolchains:all",
-    dev_dependency = True,
-)
```

Comment:
by doing this we can't make use of prebuilt protoc which increases the build time, please revert this change and bring //toools/toolchains/BUILD.bazel again